README.md

```diff
@@ -62,30 +62,30 @@ documentation, please see the [GoDoc Page](http://godoc.org/github.com/gofrs/uui
 package main
 
 import (
-	"fmt"
+	"log"
+
 	"github.com/gofrs/uuid"
 )
 
-func main() {
-	// Creating UUID Version 4
-	// panic on error
-	u1 := uuid.Must(uuid.NewV4())
-	fmt.Printf("UUIDv4: %s\n", u1)
+// Create a Version 4 UUID, panicking on error.
+// Use this form to initialize package-level variables.
+var u1 = uuid.Must(uuid.NewV4())
 
-	// or error handling
+func main() {
+	// Create a Version 4 UUID.
 	u2, err := uuid.NewV4()
 	if err != nil {
-		fmt.Printf("Something went wrong: %s", err)
-		return
+		log.Fatalf("failed to generate UUID: %v", err)
 	}
-	fmt.Printf("UUIDv4: %s\n", u2)
+	log.Printf("generated Version 4 UUID %v", u2)
 
-	// Parsing UUID from string input
-	u2, err := uuid.FromString("6ba7b810-9dad-11d1-80b4-00c04fd430c8")
+	// Parse a UUID from a string.
+	s := "6ba7b810-9dad-11d1-80b4-00c04fd430c8"
+	u3, err := uuid.FromString(s)
 	if err != nil {
-		fmt.Printf("Something went wrong: %s", err)
+		log.Fatalf("failed to parse UUID %q: %v", s, err)
```

Comment:
Why `%v` instead of `%s`? I think we'd want the error message here, which would be `%s`.

Reply:
I'm not sure `%s` for errors is right. The documentation for `%s` says "the uninterpreted bytes of the string or slice". AFAIK, `%v` for errors is interpreted as the result of `err.Error()`.

Reply:
Looks like `%v` and `%s` behave the same. TIL. 👍